spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -488,12 +500,21 @@ def setup(self):
 
         # --- Setting options menu
         options_menu = self.get_options_menu()
+
+        self.console_environment_menu = self.create_menu(
+            IPythonConsoleWidgetOptionsMenus.EnvironmentConsoles,
+            _('New console in environment'))
+        self.console_environment_menu.setStyleSheet(
+            "QMenu { menu-scrollable: 1; }"
+        )
```

Comment:
Please use `qstylizer` for this (you can find examples in `utils/stylesheet.py`). We prefer not to use css as docstrings because they are hard to modify/maintain.